docs/autotest.md

```diff
@@ -56,8 +56,11 @@ virtual machines.
 The first node is the Kubernetes master without
 persistent memory.
 The other three nodes are worker nodes with one emulated 32GB NVDIMM each.
-After the cluster has been formed, `make start` adds `storage=pmem` label
-to the worker nodes and deploys the PMEM-CSI driver.
+After the cluster has been formed, `make start` adds NFD to label
```

Comment:
'adds NFD' sounds not appropriate to me, what about something line 'starts NFD'? And a link to NFD project seems appropriate here.

Reply:
Okay, a later part of the documentation added the link to NFD, so it is fine to omit it here.

Reply:
"`make start` starts NFD" stutters.

How about "`make start` installs NFD"?

Reply:
> How about "`make start` installs NFD"?

Sounds good.